aiofirebase/__init__.py

```diff
@@ -35,23 +35,23 @@ def __init__(self, base_url, auth=None, loop=None):
         """Gracefully close the session."""
         await self._session.close()
 
-    async def get(self, *, path=None, params=None):
+    async def get(self, *, path=None, params={}):
```

Comment:
This should be None as you should not have mutable default arguments - here's a good write up of the side effects (https://pythonconquerstheuniverse.wordpress.com/category/python-gotchas/)